specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/ServerAzureADAdministrators.json

```diff
@@ -0,0 +1,507 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-02-02-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/enableAzureADOnlyAuthentication": {
```

Comment:
by making this a POST action instead of a configuration that can be set with PUT on the server (or similar) this means users cannot configure ADOnlyAuth with a template.  This should be avoided unless absolutely necessary.  Why can't this be modeled as a property on a resource that can be modified with a PUT/PATCH?

Reply:
We do not have patch in SQL. 
Since this is stand along API. and need RBAC on it. SQL API experts suggestion is to use POST. 
Other APIs in SQL have been exposed in similar fashion. 
@jamestao can please take a look.

Reply:
Hello @pilor , I have made the API changes , please take a look as possible.